domains/gxlpes.json

```diff
@@ -1,11 +1,12 @@
-
-    {
-      "owner": {
-        "username": "gxlpes",
+{
+    "description": "Website portfolio",
+    "repo": "https://github.com/<gxlpes>/<gxlpes>.github.io",
```

Comment:
You do not need to add `<>` around the repository name, it was to indicate a placeholder.
This isn't requied but would be nice if you correct this.

Reply:
Changed.

I suggest to change the [docs](https://www.is-a.dev/docs/github_pages/) as the email address does not have a placeholder, so it can be misleading for some users.